src/order/rel_classes.lean

```diff
@@ -233,6 +234,13 @@ instance empty_relation.is_well_order [subsingleton α] : is_well_order α empty
   trans        := λ a b c, false.elim,
   wf           := ⟨λ a, ⟨_, λ y, false.elim⟩⟩ }
 
+@[priority 100]
+instance is_empty.is_well_order [is_empty α] (r : α → α → Prop) : is_well_order α r :=
+{ trichotomous := is_empty_elim,
+  irrefl       := is_empty_elim,
+  trans        := is_empty_elim,
+  wf           := ⟨is_empty_elim⟩ }
```

Comment:
Does this also work as follows?
```suggestion
  wf           := is_empty_elim }
```

Reply:
If not, it might be worth adding a `well_founded_of_is_empty` lemma (with proof `⟨is_empty_elim⟩`)

Reply:
I've added said lemma.